project/Dependencies.scala

```diff
@@ -21,8 +21,7 @@ object Dependencies {
   // needs to be inline with the aeron version
   val agronaVersion = "1.0.8"
   val nettyVersion = "3.10.6.Final"
-  val jacksonVersion = "2.10.1"
-  val jacksonDatabindVersion = "2.10.1"
+  val jacksonVersion = "2.10.0"
```

Comment:
yes, we can introduce that again if they start with micro-patches of databind

Reply:
But it should be 2.10.1 no?

Reply:
good catch 😄

Reply:
doh :hushed: